DigitalRain/Columns/ColumnSpace.cs

```diff
@@ -10,12 +10,17 @@ public class ColumnSpace
         private Column _column;
         private float _positionY;
 
-        public ColumnSpace(Column column, float positionY)
+        public ColumnSpace(Column column, int number, float positionY)
         {
             _column = column;
+            RowNumber = number;
             _positionY = positionY;
         }
 
+        public int RowNumber { get; private set; }
+        public int ColumnNumber { get { return _column.Number; } }
+        public (int, int) Coordinates { get { return (RowNumber, ColumnNumber); } }
```

Comment:
Just FYI, in Monogame you can use a `Point` to define an int32 X & Y value. Maybe makes more sense than an (int, int)?

https://docs.monogame.net/api/Microsoft.Xna.Framework.Point.html#Microsoft_Xna_Framework_Point__ctor_System_Int32_System_Int32_

Reply:
```suggestion
        public int RowNumber { get; private set; }
        public int ColumnNumber { get { return _column.Number; } }
        public Point Coordinates { get { return new Point(RowNumber, ColumnNumber); } }
```